src/context/ChannelContext.tsx

```diff
@@ -0,0 +1,249 @@
+import React, { PropsWithChildren, useContext } from 'react';
+
+import type {
+  Attachment,
+  Channel,
+  Message,
+  MessageResponse,
+  Mute,
+  ChannelState as StreamChannelState,
+  StreamChat,
+  UpdateMessageAPIResponse,
+  UserResponse,
+} from 'stream-chat';
+import type {
+  Data as EmojiMartData,
+  NimbleEmojiIndex,
+  NimbleEmojiProps,
+  NimblePickerProps,
+} from 'emoji-mart';
```

Comment:
on RN the only things that were out of abc order were imports from react/react-native and those went first, everything else, including our stream-chat stuff, was abc ordered

Reply:
oh got it, thought we were giving stream stuff precedence but i'll swap em so the libs are consistent